src/lib/api/services/ProofsService.ts

```diff
@@ -26,7 +26,7 @@ export class ProofsService {
     proofId: string,
     includeProof: boolean = true,
     includePublic: boolean = true,
-    includeSmartContractCalldata: boolean = true,
+    includeSmartContractCalldata: boolean = false,
```

Comment:
This had been a manual change that was out of sync with the upstream API. We should change the OpenAPI spec to reflect it here.